packages/tenant-process/src/services/tenantService.ts

```diff
@@ -558,4 +636,33 @@ async function revaluateTenantKind({
   return updatedTenant;
 }
 
+async function revokeCertifiedAttribute(
+  tenant: Tenant,
+  readModelService: ReadModelService,
+  attributeId: AttributeId
+): Promise<Tenant> {
+  const updatedTenant: Tenant = {
+    ...tenant,
+    updatedAt: new Date(),
+    attributes: tenant.attributes.map((attr) =>
+      attr.id === attributeId
+        ? {
+            ...attr,
+            revocationTimestamp: new Date(),
+          }
+        : attr
+    ),
+  };
+
+  const updatedKind = await getTenantKindLoadingCertifiedAttributes(
```

Comment:
as per recent conversations, this should be saved in a different event (but in the same transaction as the revocation event)